src/integrals/diskdf_integrals.cc

```diff
@@ -510,7 +510,8 @@ void DISKDFIntegrals::gather_integrals() {
         }
     }
     df_->set_memory(static_cast<size_t>(mem));
-    df_->set_method("DIRECT");
```

Comment:
Could you elaborate on why you commented this line?

Reply:
Yes, I am actually testing it now (why it has not been marked as Ready to go).